idx/cassandra/cassandra.go

```diff
@@ -266,8 +266,14 @@ func (c *CasIdx) AddOrUpdate(data *schema.MetricData, partition int32) {
 	}
 
 	if updateIdx {
+		// set the new metricdef based on the data, however if this data will go back in time
+		// it'll be rejected data, and we should stick to the higher LastUpdate we already had.
 		def := schema.MetricDefinitionFromMetricData(data)
+		if inMemory && existing.LastUpdate > def.LastUpdate {
```

Comment:
just to make sure i understand that right... if we're processing the backlog then MT will come up, read the index from cassandra, and then read all the insertions from kafka and apply them. if other nodes have already updated the cassandra index then the `LastUpdate` property will have been set by them and that's why it will be ahead of the data coming in from Kafka, right?